From 662a4312adb125da54ca40060ab7378f60f020cb Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Sat, 3 Dec 2011 13:05:37 -0500
Subject: [PATCH 028/859] staging: lttng: TODO update: lttng reported to work
 fine on -rt now

The patch "lttng: Fix recent modifications to string_from_user
operation" has been confirmed to fix the corrupted trace problem
experienced on -rt kernel by the original bug reporter. Remove the
entry from the LTTng TODO list.

Reported-by: Yannick Brosseau <yannick.brosseau@gmail.com>
Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/lttng/TODO | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/staging/lttng/TODO b/drivers/staging/lttng/TODO
index 3fdc5e6..5e3a581 100644
--- a/drivers/staging/lttng/TODO
+++ b/drivers/staging/lttng/TODO
@@ -35,12 +35,6 @@ A) Cleanup/Testing
 	   depend on the producer to push the reader position.
 	   Contact: Julien Desfossez <julien.desfossez@polymtl.ca>
 
-	4) Test latest -rt kernel support.
-	   There has been report of corrupted traces when tracing a
-	   3.0.10-rt27 in the area of access_ok() system call event.
-	   Still has to be investigated. Cannot be reproduced with
-	   mainline kernel.
-	   Contact: Yannick Brosseau <yannick.brosseau@polymtl.ca>
 
 B) Features
 
-- 
1.8.1.4

